apps/web-giddh/src/app/invoice/proforma/proforma-list.component.html

```diff
@@ -108,13 +89,76 @@
                 </div>
             </div>
         </div>
+        <div class="d-flex mt-1">
+            <div *ngIf="voucherType === 'proformas'" class="btn-wrapper mr-15">
+                <button class="btn btn-pink" style="margin: 0 !important;" (click)="goToInvoice('proformas')">+ New
+                    Proforma
+                </button>
+            </div>
+
+            <div *ngIf="voucherType === 'estimates'" class="btn-wrapper mr-15">
+                <button class="btn btn-pink" style="margin: 0 !important;" (click)="goToInvoice('estimates')">+ New
+                    Estimate
+                </button>
+            </div>
+            <div *ngIf="selectedItems.length === 1" style="height: 30px;">
+
+                <div class="btn-group" dropdown #dp="bs-dropdown"
+                    style="margin-bottom: 10px;display: inline-block !important;">
```

Comment:
remove inline CSS in the all files

Reply:
I am not added this inline css, may be its spacing changes which is created by code formatting.